google/cloud/security/common/data_access/bucket_dao.py

```diff
@@ -48,3 +56,36 @@ def get_buckets_by_project_number(self, resource_name,
         rows = self.execute_sql_with_fetch(
             resource_name, buckets_sql, None)
         return [row['bucket_name'] for row in rows]
+
+    def get_buckets_acls(self, resource_name, timestamp):
+        """Select the bucket acls from a bucket acls snapshot table.
+
+        Args:
+            resource_name: String of the resource name.
+            timestamp: String of timestamp, formatted as YYYYMMDDTHHMMSSZ.
+
+        Returns:
+            List of bucket acls.
+
+        Raises:
+            MySQLError: An error with MySQL has occurred.
+        """
+        bucket_acls = {}
+        cnt = 0
+        try:
+            cursor = self.conn.cursor()
+            cursor.execute(select_data.BUCKET_ACLS.format(timestamp))
```

Comment:
not super high priority (i.e. you can do this in a separate PR), but there's a function in the base dao class, "execute_sql_with_fetch(resource_name, query, param_tuple)", that will reduce some of the connection/cursor boilerplate code.